tests/test_diff.py

```diff
@@ -594,335 +561,85 @@ def test_diff_polygons(output_format, data_working_copy, geopackage, cli_runner)
                 len(odata["sno.diff/v1"]["nz_waca_adjustments"]["featureChanges"]) == 4
             )
             assert odata == {
-                "sno.diff/v1": {
-                    "nz_waca_adjustments": {
-                        "metaChanges": {},
-                        "featureChanges": [
+                'sno.diff/v1': {
```

Comment:
Need some indicator of what sort of geometry encoding is being used ([see](https://github.com/koordinates/sno/issues/62#issuecomment-620507966))

Reply:
I don't think we do. (Let's discuss/resolve this on that thread^ on #62 though)